packages/mariadb-connection-module/package.json

```diff
@@ -0,0 +1,30 @@
+{
+  "name": "mariadb-connection-module",
+  "version": "1.0.0",
+  "description": "> TODO: description",
+  "author": "jinfanv <infanpearl@gmail.com>",
+  "homepage": "https://github.com/platzily/platzily-monolith#readme",
+  "license": "ISC",
+  "main": "src/index.js",
+  "directories": {
+    "lib": "lib",
+    "test": "__tests__"
+  },
+  "files": [
+    "lib"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/platzily/platzily-monolith.git"
+  },
+  "scripts": {
+    "test": "echo \"Error: run tests from root\" && exit 1"
+  },
+  "bugs": {
+    "url": "https://github.com/platzily/platzily-monolith/issues"
+  },
+  "dependencies": {
+    "knex": "^0.95.8",
```

Comment:
Can we use exact versions? We can acchieve this removing the `^` on the versions. This way we gain control of the versions we're using.